Makefile

```diff
@@ -19,6 +19,9 @@ CCEND=\033[0m
 help: ## Show this help.
 	@awk 'BEGIN {FS = ":.*?## "} /^[a-zA-Z_-]+:.*?## / {sub("\\\\n",sprintf("\n%22c"," "), $$2);printf "\033[36m%-20s\033[0m %s\n", $$1, $$2}' $(MAKEFILE_LIST)
 
+gen-cve-feed: ## Generates content from GCS bucket with official cve data.
+	scripts/security/gen-cve-feed.sh
+
```

Comment:
Hugo can fetch this JSON file itself - see https://gohugo.io/templates/data-templates/

I think I'd prefer that, so we're not modifying the website source during a build.

Reply:
Yes good call. It took some work, but got this working without build changes